src/tabs/Tabs.tsx

```diff
@@ -119,11 +122,39 @@ export class Tabs extends PureComponent<TabsProps, TabsState> {
     }
   }
 
+  moveHighlight = (activeTabId: string) => {
+    const currentTab = this.state.tabIdToRefs.get(activeTabId).current
+    const tabBounds = currentTab.getBoundingClientRect()
+
+    // tablist parent needed to get the height of the whole component
+    // (tabs can have different heights)
+    const parentTablist = currentTab.closest('.kirk-tablist')
+    const parentBounds = parentTablist.getBoundingClientRect()
+
+    // adapt to current tab width
+    this.highlightRef.current.style.width = `${tabBounds.width}px`
+    // top position, 1px = wrapper gray border
+    this.highlightRef.current.style.top = `${parentBounds.height - 1}px`
```

Comment:
Can you get that `1px` dynimacally? If not, add a const above.

Reply:
I changed it (changed the parent element I used)